src/server/podcastRssFeed.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import config from 'config';
+import { fetchSeries } from '../util/audioApi';
+
+const podcastRssFeed = async (seriesId: number): Promise<string> => {
+  try {
+    const series = await fetchSeries(seriesId, 'nb');
+    const podcastUrl = `${config?.ndlaFrontendDomain}/podcast/${series.id}`;
+    const ownerEmail = 'support+podcast@ndla.no';
+
+    const authors = series.episodes
+      .flatMap(ep => {
+        return ep.copyright.creators
+          .concat(ep.copyright.processors)
+          .concat(ep.copyright.rightsholders);
+      })
+      .map(a => a.name);
+
+    const uniqueAuthors = Array.from(new Set(authors)).map(
+      a => `<itunes:author>${a}</itunes:author>`,
+    );
+
+    const description = `
+    <description>
+      <![CDATA[
+      ${series.description.description}
+      ]]>
+    </description>
+    `;
+
+    const episodes = series.episodes.map(episode => {
+      const episodeLink = `${podcastUrl}#episode-${episode.id}`;
+      const episodeGUID = `NDLA-${config.ndlaEnvironment}-${episode.id}`;
```

Comment:
For prod trengs ikkje miljøvariabel. Det gir bra meining med test og staging, men for prod funker 'NDLA-1' og så videre.

Reply:
det var smart :+1: